lib/Backend/ObjTypeSpecFldInfo.cpp

```diff
@@ -940,13 +1077,23 @@ ObjTypeSpecFldInfo* ObjTypeSpecFldInfo::CreateFrom(uint id, Js::PolymorphicInlin
             types[i].t = RecyclerNew(recycler, JITType);
             __analysis_assume(localTypes[i] != nullptr);
             JITType::BuildFromJsType(localTypes[i], types[i].t);
+
+            if (depolymorphize)
+            {
+                polymorphicInfoArray[i] = localPolymorphicInfo[i];
+            }
```

Comment:
nit: we could use CopyArray (as used to copy localFixedFieldInfoArray) in the previous if(depolymorphize) branch.

Reply:
Good point. I'll make a separate commit to do that. Thanks.